desktop/assets/auctions.coffee

```diff
@@ -11,6 +11,7 @@ routes =
   '''
   /sale/.*
   ^/auction/[^/]+/?$
+  /auction/.*/confirm-registration
```

Comment:
Can you put a `FIXME` on that regex describing whats going on and how it catches subroutes? It's dangerous :)